### PR TITLE
Draft NEP 28: Standardization of the Smallest Unit of GAS

### PR DESCRIPTION
**Pull request by Jim8y**
This NEP proposes to standardize the terminology for the smallest unit of GAS by naming it "Datoshi." This change aims to eliminate ambiguity in NEO's financial transactions and further refine the clarity of GAS as a divisible currency unit.

The term "Datoshi" is proposed as the name for the smallest unit of GAS. The relationship between GAS and Datoshi is defined as follows:
- 1 Datoshi = 1e-8 GAS
- 1 Kdatoshi (thousand Datoshi) = 1e-5 GAS
- 1 Mdatoshi (million Datoshi) = 1e-2 GAS

In equivalency:
- 100,000,000 Datoshi = 100,000 Kdatoshi = 100 Mdatoshi = 1 GAS

This specification introduces a clear, scalable denomination system for GAS that aids in transactions, contracts, and discussions within the NEO ecosystem.

### CODE REVIEW

File: nep-26.mediawiki [hunk 1]
```diff
@@ -0,0 +1,44 @@
+NEP: 26
```
Comment (shargon) [inline nep-26.mediawiki:1]: 27?

File: nep-27.mediawiki [hunk 2]
```diff
@@ -0,0 +1,44 @@
+NEP: 27
+Title: Standardization of the Smallest Unit of GAS
+Author: Jimmy Liao <jimmy@r3e.network>
+Type: Standard
+Status: Draft
+Created: 2024-05-20
+Requires (*optional): None
+Replaces (*optional): None
```
Comment (shargon) [inline nep-27.mediawiki:8]: We can remove it

Comment (Jim8y) [inline nep-27.mediawiki:8]:
```suggestion
```

**Review by shargon — APPROVED** [review]

File: nep-27.mediawiki [hunk 3]
```diff
@@ -0,0 +1,42 @@
+NEP: 27
```
Comment (AnnaShaleva) [inline nep-27.mediawiki:1]: Likely should be 28. Ref. https://github.com/neo-project/proposals/pull/169#discussion_r1608980280.

File: nep-27.mediawiki [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Title: Standardization of the Smallest Unit of GAS
+Author: Jimmy Liao <jimmy@r3e.network>
+Type: Standard
+Status: Draft
+Created: 2024-05-20
+
+==Abstract==
+
+This NEP proposes to standardize the terminology for the smallest unit of GAS by naming it "Datoshi." This change aims to eliminate ambiguity in NEO's financial transactions and further refine the clarity of GAS as a divisible currency unit.
```
Comment (AnnaShaleva) [inline nep-27.mediawiki:10]: > in NEO's financial transactions

It's "Neo" in the official documentation, we likely should follow.

File: nep-27.mediawiki [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+This proposal does not introduce any technical incompatibilities but will require updates to documentation, user interfaces, and educational materials to reflect the new terminology.
+
+==Test Cases==
+
+N/A
+
+==Implementation==
+
+Prior to final approval of this NEP, updates will be prepared for all NEO platform documentation and user interfaces to incorporate the term "Datoshi." This preparation will ensure a smooth transition upon formal acceptance of the proposal.
```
Comment (AnnaShaleva) [inline nep-27.mediawiki:42]: We probably should update it everywhere before the standard merge, and then post a couple of links here.

**Review by shargon — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Review by AnnaShaleva — COMMENTED** [review]
All conversations are fixed from my side, this PR is ready to be merged. But still, I'm not a big fan of this change, so can't help with approvals gain.

**Comment by Jim8y** [comment]
> All conversations are fixed from my side, this PR is ready to be merged. But still, I'm not a big fan of this change, so can't help with approvals gain.

@AnnaShaleva Decisions should be made, either disapproval or approval. 4 votes plus an author, it like 5 approvals to merge, if 2 core-devs are like `i am cool with it, but i am not going to approve`, then the system wont work. 

Same thing happened before as @vncoelho also told me that he has no preference to the prs or proposals, so he is not disapprove or approve it..... Come on, core-devs are supposed to make decissions, you dont approve already mean a disapprove. If we all like this, then only proposals or prs of your own benifits or interests can be processed.

**Review by AnnaShaleva — APPROVED** [review]
Standard is needed? Yes. The thing that I don’t understand is why should we change the Satoshi term that is used across the codebase (both C# and Go) from the very beginning of the Neo project.

Anyway, I’ve already approved the https://github.com/neo-project/neo/pull/3241, and we have https://github.com/nspcc-dev/neo-go/issues/3461, and arguing here seems not to be logical from my side.

**Review by NGDAdmin — APPROVED** [review]

**Pull request merged**